include/zenoh-pico/api/constants.h

```diff
@@ -133,24 +135,25 @@ typedef enum
  */
 typedef enum
 {
-    Z_QUERYABLE_ALL_KINDS = 0x01,  // 1 << 0
-    Z_QUERYABLE_STORAGE = 0x02,    // 1 << 1
-    Z_QUERYABLE_EVAL = 0x04        // 1 << 2
+    Z_QUERYABLE_ALL_KINDS = 0x01, // 1 << 0
+    Z_QUERYABLE_STORAGE = 0x02,   // 1 << 1
+    Z_QUERYABLE_EVAL = 0x04       // 1 << 2
 } z_queryable_kind_t;
 
 /**
  * Consolidation mode values.
  *
  * Enumerators:
- *     Z_CONSOLIDATION_MODE_FULL: Guaranties unicity of replies. Optimizes bandwidth.
- *     Z_CONSOLIDATION_MODE_LAZY: Does not garanty unicity. Optimizes latency.
- *     Z_CONSOLIDATION_MODE_NONE: No consolidation.
+ *     Z_CONSOLIDATION_MODE_NONE: No consolidation is applied. Replies may come in any order and any number.
+ *     Z_CONSOLIDATION_MODE_MONOTONIC: It guarantees that any reply will be monotonic in time w.r.t.
+ *          the previous received replies. It does not garantuee unicity of replies. Optimizes latency.
```

Comment:
Does it not guarantee unicity of replies with the same key-timestamp tuple? I think zenoh-rs does, and if not, it's an easy fix that shouldn't change the performances :)

Reply:
It works exactly as in `zenoh-rs`. It seems it is simply poorly worded. What about: 
> Z_CONSOLIDATION_MODE_MONOTONIC: It guarantees that any reply for a given key expression will be monotonic in time w.r.t. the previous received replies for the same key expression. I.e., for a key expression A it is guaranteed that two replies received at t1 and t2 will have timestamp ts2 > ts1.  It optimizes latency.